Codeinwp/neve
# Feat/issue 3071

### Summary
Changed the Custom HTML component for HFG to use the WP wysiwig editor.
It uses the editor provided by WordPress and only initializes the editor once the component is added and the section is opened for editing.

### Will affect visual aspect of the product
YES

### Screenshots
![image](https://user-images.githubusercontent.com/23024731/135839484-bc1b5bc7-54c1-47e2-a09c-66bca5e9e886.png)

![image](https://user-images.githubusercontent.com/23024731/135839508-cbc478f3-ba7d-46c4-b936-8b9720caf762.png)

### Test instructions
1. Add a new and/or use an existing HTML component inside Header, Page Header or Footer
2. Check that the dynamic magic tags are still working as before
3. Check that editor works as expected


Closes #3071.
